### DISCUSSION TITLE
Length matching for Transceivers and HSIO signals on FMC+ pin

### DISCUSSION THREAD

**Discussion by Garripa** [ArrowElectronics/Agilex-5#2] [General]
Could someone from the board development team confirm the tolerance on the length matching for the transceiver pins as well as the HSIO pins to FMC+ connector. 
I would also like to know if the transceivers from the same banks are length matched or not. 
Does length matching change for HSIO signals for the Top and the bottom sided of the bank 2A ?

**Comment by skravats** [comment]
Apologies for the delay in replying. Please see the link to the file below.

https://github.com/ArrowElectronics/Agilex-5/blob/main/images/promo3/TL-TEI0185-03.xls
